clients/java/signalr/src/main/java/Transport.java

```diff
@@ -3,8 +3,8 @@
 
 public interface Transport {
     void start() throws InterruptedException;
-    void send(String message);
+    void send(String message) throws Exception;
```

Comment:
Doesn't it require catching base exception? Can/should this Exception be of more specific type (possibly even SignalRException)?